app/main.dev.js

```diff
@@ -670,11 +671,22 @@ ipcMain.on(Messages.LOAD_PROJECT_LOG_REQUEST, async (event, project) => {
 ipcMain.on(
   Messages.UPDATE_PROJECT_REQUEST,
   // eslint-disable-next-line prettier/prettier
-  async (event, projectPath, actionType, entityType, entityKey, title, description, details, level, user) => {
+  async (
```

Comment:
I'm okay to accept the linter suggestion here, but we should remove the eslint-disable line above